MAUI/Scheduler/migration.md

```diff
@@ -780,6 +780,74 @@ this.Content = scheduler;
 <td>{{'[DayFormat](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerViewHeaderSettings.html#Syncfusion_Maui_Scheduler_SchedulerViewHeaderSettings_DayFormat)'| markdownify }} (From ViewHeaderSettings of MonthView class) </td>
 <td>Gets or sets a view header day format of the SfScheduler.</td></tr>
  </table> 
+ 
+The following code example, explains how to initialize the properties of the Xamarin SfSchedule and .NET MAUI SfScheduler class.
```

Comment:
The following code example, explains how to configure the month view settings in Xamarin SfSchedule and .NET MAUI SfScheduler.